Workbooks/Communication Services/Overview/Overview.workbook

```diff
@@ -0,0 +1,1030 @@
+            "linkLabel": "Chat",
+            "subTarget": "chat",
+            "style": "link"
+          }
+        ]
+      },
+      "name": "nav_menu"
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 1,
+            "content": {
+              "json": "The Overview tab displays counts over time of your key communications services' activity."
+            },
+            "name": "overview-text"
+          },
+          {
+            "type": 9,
+            "content": {
+              "version": "KqlParameterItem/1.0",
+              "parameters": [
+                {
+                  "id": "0cfaad60-33d7-4643-89e3-17cb0b99eb03",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "plot_type",
+                  "label": "Plot type",
+                  "type": 2,
+                  "description": "Select the type of plot to visualize the data",
+                  "isRequired": true,
+                  "typeSettings": {
+                    "additionalResourceOptions": []
+                  },
+                  "jsonData": "[\r\n    {\"value\": \"columnchart\", \"label\": \"Bar chart\", \"selected\":true},\r\n    {\"value\": \"areachart\", \"label\": \"Area chart\"}\r\n]",
+                  "timeContext": {
+                    "durationMs": 86400000
+                  }
+                }
+              ],
+              "style": "pills",
+              "queryType": 0,
+              "resourceType": "Microsoft.Communication/CommunicationServices"
+            },
+            "name": "overview-params"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "ACSCallSummary\r\n| where CallStartTime {time_range:query}\r\n| distinct CorrelationId, CallStartTime, CallType\r\n| extend time_span = floor(CallStartTime, {time_granularity})\r\n| make-series Calls=count() default=0 on CallStartTime from {time_range:start} to {time_range:end} step {time_granularity} by time_span\r\n| render {plot_type}",
+              "size": 0,
+              "title": "Voice and video call counts",
+              "queryType": 0,
+              "resourceType": "Microsoft.Communication/CommunicationServices"
+            },
+            "customWidth": "50",
+            "name": "voice-video-overview"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "ACSChatIncomingOperations\r\n| where TimeGenerated {time_range:query}\r\n//| distinct CorrelationId, CallStartTime, CallType\r\n| extend time_span = floor(TimeGenerated, {time_granularity})\r\n| make-series ChatOperations=count() default=0 on TimeGenerated from {time_range:start} to {time_range:end} step {time_granularity} by time_span\r\n| render {plot_type}",
```

Comment:
![image](https://user-images.githubusercontent.com/7664931/140432791-ca36498f-c0ae-464e-a17d-dee70ce9f853.png)
Just fyi, there's a dropdown where you can select a time range parameter fora q uery item, then you don't need that Where TimeGenerated {time_Range:querey} line

Reply:
Got it, I just wanted a single parameter that updated the whole workbook, but I will keep this in mind in the future. Thanks!